MobiFlight/Board.cs

```diff
@@ -242,7 +242,7 @@ public class UsbDriveSettings
         public String VolumeLabel;
     }
 
-    public class Board
+    public class Board : IMigrateable
```

Comment:
All JSON-based objects need to implement `IMigrateable` now, even if `Migrate()` is empty, so generics work properly when loading and validating the JSON.